dlt/destinations/impl/lancedb/configuration.py

```diff
@@ -0,0 +1,102 @@
+import dataclasses
+from typing import Optional, Final, Literal, ClassVar, List
+
+from dlt.common.configuration import configspec
+from dlt.common.configuration.specs.base_configuration import (
+    BaseConfiguration,
+    CredentialsConfiguration,
+)
+from dlt.common.destination.reference import DestinationClientDwhConfiguration
+from dlt.common.typing import TSecretStrValue
+from dlt.common.utils import digest128
+
+
+@configspec
+class LanceDBCredentials(CredentialsConfiguration):
+    uri: str = "./.lancedb"
```

Comment:
is this a file or a folder? why do we need the "." prefix?

Reply:
I remove the prefix so now its just `.lancedb` . This good?